packages/wrangler/src/__tests__/queues.test.ts

```diff
@@ -247,6 +259,40 @@ describe("wrangler", () => {
 			"
 		`);
 			});
+
+			it("should send send queue settings with delivery delay", async () => {
+				const requests = mockCreateRequest("testQueue", { delivery_delay: 10 });
+				await runWrangler("queues create testQueue --delivery-delay=10");
+				expect(std.out).toMatchInlineSnapshot(`
+					"Creating queue testQueue.
+					Created queue testQueue."
+			  `);
+				expect(requests.count).toEqual(1);
+			});
+
+			it("should send send queue settings with 0 delivery delay when noDeliveryDelay is set", async () => {
```

Comment:
Ideally the request would just omit the delivery_delay (or set it to `undefined`) instead of `0` IMO